isis/src/base/objs/ProcessExportPds4/ProcessExportPds4.cpp

```diff
@@ -1126,32 +1165,73 @@ namespace Isis {
-      }
+    
+    // Add Lat/Lon range
+    double maxLon, minLon, maxLat, minLat; 
+    InputCubes[0]->latLonRange(minLat, maxLat, minLon, maxLon); 
+    
+    xmlPath.clear();
+    xmlPath << "Product_Observational"
+            << "Observation_Area" 
+            << "Discipline_Area"
+            << "cart:Cartography" 
+            << "cart:Spatial_Domain"
+            << "cart:Bounding_Coordinates";
+    QDomElement boundingCoordElement = getElement(xmlPath, baseElement);
+    QDomElement eastElement = boundingCoordElement.firstChildElement("cart:east_bounding_coordinate");
+    QDomElement westElement = boundingCoordElement.firstChildElement("cart:west_bounding_coordinate");
+    QDomElement northElement = boundingCoordElement.firstChildElement("cart:north_bounding_coordinate");
+    QDomElement southElement = boundingCoordElement.firstChildElement("cart:south_bounding_coordinate");
+
+    // Translation files currently handles Positive West case where east = min, west = max
+    // so if positive east, swap min/max
+    if(QString::compare(lonDir, "PositiveEast", Qt::CaseInsensitive) == 0) {
+      // west min, east max
+      PvlToXmlTranslationManager::resetElementValue(eastElement, toString(maxLon), "deg");
+      PvlToXmlTranslationManager::resetElementValue(westElement, toString(minLon), "deg");
     }
+    else {
+      PvlToXmlTranslationManager::resetElementValue(eastElement, toString(minLon), "deg");
+      PvlToXmlTranslationManager::resetElementValue(westElement, toString(maxLon), "deg");
+    }
+
+    PvlToXmlTranslationManager::resetElementValue(northElement, toString(maxLat), "deg");
+    PvlToXmlTranslationManager::resetElementValue(southElement, toString(minLat), "deg");
+
+    // longitude_of_central_meridian and latitude_of_projection_origin need to be converted to floats.
+    xmlPath.clear();
+    xmlPath << "Product_Observational"
+            << "Observation_Area" 
+            << "Discipline_Area"
+            << "cart:Cartography" 
+            << "cart:Spatial_Reference_Information"
+            << "cart:Horizontal_Coordinate_System_Definition"
+            << "cart:Planar"
+            << "cart:Map_Projection";
+
+    // The following is necessary because the full xmlPath differs depending on the projection used.
+    QDomElement projectionElement = getElement(xmlPath, baseElement);
+    QDomElement tempElement = projectionElement.firstChildElement();
+    QDomElement nameElement = tempElement.nextSiblingElement();
+
+    QDomElement longitudeElement = nameElement.firstChildElement("cart:longitude_of_central_meridian"); 
+    QDomElement originElement = nameElement.firstChildElement("cart:latitude_of_projection_origin");
+
+    double longitudeElementValue = longitudeElement.text().toDouble(); 
+    double originElementValue = originElement.text().toDouble(); 
+    QString toset1 = QString::number(longitudeElementValue, 'g');
```

Comment:
If anyone has any better idea for how to do this, let me know. Forcing it to be a float somehow didn't force it to output with the required ".0" so I just added it this way.

Reply:
Can you use the precision argument from [Qstring::number](https://doc.qt.io/qt-5/qstring.html#number-6)?

Reply:
Amusingly, I went down this odd rabbit hole in the first place because 'g' removes trailing 0's and I didn't realize 'f' didn't.